docs/getting-started/apple/screenshots.md

```diff
@@ -1,8 +1,238 @@
+
+# Put Your Screenshots Into Device Frames
+
+_frameit_ helps you beautify your screenshots with devices frames and text by running one simple command. It provides support for:
+
+- Multiple device types
+- Portrait and landscape orientations
+- Black and silver devices
+- Setting a background color and decorating the image with text
+
+![frameit results](assets/frameit-results.png)
+
+## Dependencies
+
+### Installing ImageMagick
+
+_frameit_ depends on a tool called `imagemagick` to do image manipulation. The easiest way to install it is through [homebrew](http://brew.sh/):
+
+```sh
+brew install libpng jpeg imagemagick
+```
+
+### Troubleshooting ImageMagick
+
+If you have installed _imagemagick_ but are seeing error messages like:
+
+```sh
+mogrify: no decode delegate for this image format `PNG'
+```
+
+You may need to reinstall and build from source. Run:
+
+```sh
+brew uninstall imagemagick; brew install libpng jpeg; brew install imagemagick --build-from-source
+```
+
+### Setting Up Device Frames
+
+_fastlane_ uses device frames provided by Apple which need to be downloaded and installed separately for legal reasons. Running `fastlane frameit setup` can walk you through the process.
+
+**TODO: Update this when we start distributing frames ourselves.**
+
+```
+$ fastlane frameit setup
+```
+
+## Running
+
+At its simplest, you can just run:
+
+```sh
+fastlane frameit
+```
+
+A more complex example that configures a lot more about the output can be found in the [fastlane examples](https://github.com/fastlane/examples/tree/master/MindNode/screenshots) project.
+
+### Configure Using `Framefile.json`
+
+Your `Framefile.json` file should be in your `screenshots` folder, as seen in the [example](https://github.com/fastlane/examples/tree/master/MindNode/screenshots), and you can use it to define your configuration:
+
+```json
+{
+  "default": {
+    "keyword": {
+      "font": "./fonts/MyFont-Rg.otf"
+    },
+    "title": {
+      "font": "./fonts/MyFont-Th.otf",
+      "color": "#545454"
+    },
+    "background": "./background.jpg",
+    "padding": 50,
+    "show_complete_frame": false
+  },
+
+  "data": [
+    {
+      "filter": "Brainstorming",
+      "keyword": {
+        "color": "#d21559"
+      }
+    },
+    {
+      "filter": "Organizing",
+      "keyword": {
+        "color": "#feb909"
+      }
+    },
+    {
+      "filter": "Sharing",
+      "keyword": {
+        "color": "#aa4dbc"
+      }
+    },
+    {
+      "filter": "Styling",
+      "keyword": {
+        "color": "#31bb48"
+      }
+    }
+  ]
+}
+```
+
+The `show_complete_frame` value specifies whether _frameit_ should shrink the device and frame so that they show in full in the framed screenshot. If it is false, then they can hang over the bottom of the screenshot.
+
+The `filter` value is a part of the screenshot named for which the given option should be used. If a screenshot is named `iPhone5_Brainstorming.png` the first entry in the `data` array will be used.
```

Comment:
I'd love to move all the advanced documentation to somewhere else to be honest. We haven't found a great way to do so yet. This is a general issue we have to solve, I'll create a separate issue to discuss this.

This guide is a more a `Getting started with screenshots in fastlane`, and this is already quite advanced I think